docs/docs/segment-path.md

```diff
@@ -43,12 +43,12 @@ Display the current path.
 ## Mapped Locations
 
 Allows you to override a location with an icon. It validates if the current path **starts with** the value and replaces
-it with the icon if there's a match. To avoid issues with nested overrides, Oh my posh will sort the list of mapped
+it with the icon if there's a match. To avoid issues with nested overrides, oh-my-posh will sort the list of mapped
```

Comment:
The docs should always refer to Oh My Posh, to stay consistent.

Reply:
Okay -- I thought I'd seen it a few ways in the current docs, and the current "Oh my posh" didn't seem right. I'll change this.

Reply:
If we now align I do believe oh-my-posh makes more sense, but that's beyond the scope of this PR IMHO.